gnocchi/rest/__init__.py

```diff
@@ -665,6 +665,9 @@ def post(self):
         except indexer.NoSuchResource as e:
             abort(404, e)
 
+        return pecan.request.indexer.get_resource(
```

Comment:
Do you mean that the user should somehow specify if the output is needed? Could you show me an example of this?